docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.9"
```

Comment:
premenovat file je OK, aj ked hned jak ta pridas druhy tak uz to nebude fungovat a budes sa musiet vratit k povodnemu konceptu :)

Reply:
Je to podporna infrastruktura. Predpokladam ze jeden by mal stacit a pridavas len servicy. Ak budes mat ine potreby tak to refaktornes... Idealne nechat najviac jednoduche ako sa da.